mirrormaker_replication_flow.go

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2018 Aiven, Helsinki, Finland. https://aiven.io/
+
+package aiven
+
+type (
+	// MirrorMakerReplicationFlowHandler is the client which interacts with the
+	// Kafka MirrorMaker 2 ReplicationFlows endpoints on Aiven.
+	MirrorMakerReplicationFlowHandler struct {
+		client *Client
+	}
+
+	// MirrorMakerReplicationFlowRequest request used to create a Kafka MirrorMaker 2
+	// ReplicationFlows entry.
+	MirrorMakerReplicationFlowRequest struct {
+		Enabled         bool     `json:"enabled"`
+		SourceCluster   string   `json:"source_cluster"`
+		TargetCluster   string   `json:"target_cluster"`
+		Topics          []string `json:"topics"`
+		TopicsBlacklist []string `json:"topics.blacklist"`
+	}
+
+	// MirrorMakerReplicationFlowResponse represents the response from Aiven after
+	// interacting with the Kafka MirrorMaker 2 API.
+	MirrorMakerReplicationFlowResponse struct {
+		APIResponse
+		ReplicationFlows MirrorMakerReplicationFlowRequest `json:"replication_flows"`
+	}
+)
+
+// Create creates new Kafka MirrorMaker 2 Replication Flows entry.
+func (h *MirrorMakerReplicationFlowHandler) Create(project, service string, req MirrorMakerReplicationFlowRequest) (*MirrorMakerReplicationFlowResponse, error) {
+	path := buildPath("project", project, "service", service, "mirrormaker", "replication-flows")
+	bts, err := h.client.doPostRequest(path, req)
+	if err != nil {
+		return nil, err
+	}
+
+	var rsp MirrorMakerReplicationFlowResponse
```

Comment:
Create response looks to be empty. It does not contain the replication flow data as expected here.